3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -15,16 +15,35 @@ const rollDice = require('../../helpers/pokerDiceRoller');
 
 function rollTheDices() {
   const dices = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDice()
+  const promises = dices.map(dice => {
+    return rollDice(dice);
+  });
```

Comment:
```suggestion
  const promises = dices.map(rollDice);
```

This will do the same but in a more concise way.